src/MagicWormhole/Internal/Rendezvous.hs

```diff
@@ -65,44 +61,90 @@ import MagicWormhole.Internal.WebSockets (WebSocketEndpoint(..))
 -- b) (probably) either catch things or switch our stuff to exceptions
 -- c) try to figure out some testing strategy
 
--- | Run a Magic Wormhole Rendezvous client.
+
+-- | Abstract type representing a Magic Wormhole session.
+--
+-- - 'with' gets a session
```

Comment:
Does `with` still exist?  It seems like it has been removed  in favor of just `runClient`.

Reply:
It does not. Will fix before merging.